webrtc-stats.html

```diff
@@ -1617,6 +1618,18 @@ <h2>
                   packetize the resulting data.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>totalPacketSendDelay</code></dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  The total number of seconds that packets have spent buffered locally before being
+                  transmitted onto the network. Measurement starts when a packet has been produced
+                  and ends when the packet is sent. <code>totalPacketSendDelay</code> is incremented
+                  for each packet sent by this measurement for that packet.
```

Comment:
Clarify: When is the clock started, e.g. when a large frame is split up into multiple packets.
Clarify: When the clock stops, i.e. does this include delays for passing this over to the network?

Reply:
Will clarify to say something like...
Start time: When the packet was emitted from the RTP packetizer.
End time: When the packet was handed to the OS network socket.